nimbus/src/evaluator.rs

```diff
@@ -1,13 +1,14 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
 
-//! This might be where the bucketing logic can go
-//! It would be different from current experimentation tools
-//! There is a namespacing concept to allow users to be in multiple
-//! unrelated experiments at the same time.
+// ! This might be where the bucketing logic can go
```

Comment:
I think that this was done to me by a `cargo fmt`, definitely not intentional.  The comments do seem stale so I think it would probably be better to just remove them.